README.md

```diff
@@ -1 +1,129 @@
-# helloworld
\ No newline at end of file
+<a name="Hello Microverse Project"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 **HELLO MICROVERSE PROJECT** <a name="about-project"></a>
+
+**Hello Microverse Project. Jose RM.** In this project, I set up a "Hello world" repository. I applied the tools and best practices that I learned about in previous steps.
+
+## 🛠 Built With <a name="Visual Studio Code">HTML5 & CSS3</a>
+
+### Tech Stack <a name="Visual Studio Code"></a>
+
+### Key Features <a name="key-features"></a>
+
+- HTML5
+- CSS5
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Clone the repository using git, and open it with visual studio code.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd "your-folder"
+  git clone git@github.com:https://github.com/joramot/hello-world.git
+
+
+### Usage
+
+To open the project, use yout prefered explorer and open the index.html file 
+
+
+### Deployment
+
+You can deploy this project using visual studio.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 ** Jose RM **
+
+- GitHub: https://github.com/joramot
+- Twitter: https://twitter.com/el_rulis
+- LinkedIn: https://linkedin.com/in/jose-RM
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML5]**
+- **[CSS3]**
+- **[Node.js]**
```

Comment:
- So @joramot as suggested by the previous reviewer please add a "Future Features" section. I think you mistakenly added another "key features" section to your readme, so please make sure to add a Future Feature section with at least 1 feature you plan to add🎯. This will show your future clients you have a growth mindset by continuously planning & adding improvements.

Reply:
oooh !!!, that´s true!!! sorry, I will fix in one moment.